iaf-commons/src/main/java/nl/nn/adapterframework/util/DateUtils.java

```diff
@@ -33,43 +34,61 @@
 @Log4j2
 public class DateUtils {
 	public static final String fullIsoFormat          = "yyyy-MM-dd'T'HH:mm:sszzz";
+	public static final DateFormat FULL_ISO_FORMATTER = new SimpleDateFormat(fullIsoFormat);
 	public static final String shortIsoFormat         = "yyyy-MM-dd";
 
 	public static final String FORMAT_FULL_GENERIC      = "yyyy-MM-dd HH:mm:ss.SSS";
+	public static final DateFormat FULL_GENERIC_FORMATTER = new SimpleDateFormat(FORMAT_FULL_GENERIC);
 	public static final String FORMAT_MILLISECONDS	   ="######.###";
 	public static final String FORMAT_GENERICDATETIME  ="yyyy-MM-dd HH:mm:ss";
+	public static final DateFormat GENERIC_DATETIME_FORMATTER = new SimpleDateFormat(FORMAT_GENERICDATETIME);
 	public static final String FORMAT_DATE             ="dd-MM-yy";
 	public static final String FORMAT_TIME_HMS         ="HH:mm:ss";
 
-	public static String format(Date date, String dateFormat) {
-		SimpleDateFormat formatter = new SimpleDateFormat(dateFormat);
+	public static String format(Date date, DateFormat formatter) {
 		return formatter.format(date);
 	}
 
+	public static String format(Date date, String dateFormat) {
+		return format(date, new SimpleDateFormat(dateFormat));
+	}
+
 	public static String format(long date, String dateFormat) {
 		return format(new Date(date), dateFormat);
 	}
 
 	public static String format(long date) {
-		return format(new Date(date), FORMAT_FULL_GENERIC);
+		return format(new Date(date));
 	}
 	public static String format(Date date) {
-		return format(date, FORMAT_FULL_GENERIC);
+		return format(date, FULL_GENERIC_FORMATTER);
+	}
+
+	public static String formatNow() {
+		return format(new Date());
+	}
+
+	public static String formatNow(DateFormat format) {
+		return format(new Date(), format);
+	}
+
+	public static String formatNow(String format) {
+		return formatNow(new SimpleDateFormat(format));
 	}
 
 
 	/**
 	 * Get current date-time timestamp in ISO 8601 format.
 	 */
 	public static String getIsoTimeStamp() {
-		return format(new Date(), fullIsoFormat);
+		return formatNow(fullIsoFormat);
 	}
 
 	/**
 	 * Get current date-time timestamp in generic format.
 	 */
 	public static String getTimeStamp() {
-		return format(new Date(), FORMAT_GENERICDATETIME);
+		return formatNow(GENERIC_DATETIME_FORMATTER);
 	}
 
 	/**
```

Comment:
parseAnyDate mag gedeprecate worden
methodes die niet gebruikt worden mogen weg
parseToDate -> altijd DateUtils.FORMAT_FULL_GENERIC
alle statics in FULL_CAPS